src/Stuart/Client.php

```diff
@@ -31,6 +33,24 @@ public function validateJob(Job $job)
         }
     }
 
+    /**
+     * @param $city
+     * @param $type pickup or dropoff
+     * @param \DateTime $dateTime
+     *
+     * @return SchedulingSlots|object
+     */
+    public function getSchedulingSlots($city, $type, \DateTime $dateTime)
+    {
+        $apiResponse = $this->httpClient->performGet('/v2/jobs/schedules/'.$city.'/'.$type.'/'.$dateTime->format('Y-m-d'));
```

Comment:
Please can you make sure the code is not reformatted with your own rules? In the rest of the code, there as spaces before and after the concatenation char `.`. Until we don't have any guidance for formatting, we should keep how it is. We could definitely work on a proper formatter in another PR.